### DISCUSSION TITLE
Since big update anyone having ffmpeg path issues ~ Docker

### DISCUSSION THREAD

**Discussion by aivrar** [deforum-art/sd-webui-text2video#98] [General]
Hey all, I was using this quite well last week or so until I updated to the new big update and I'm having a lot of problems with ffmpeg and video creation. I'm using Docker with this script: https://github.com/AbdBarho/stable-diffusion-webui-docker

On text2vid.py for ffmpeg parameters it's set for 'find_ffmpeg_binary()' by default, but the folder isn't found. I have ffmpeg installed on C and also in my PATH etc, it was all working before before update. I copied the ffmpeg folder etc to Docker data folder and ran a direct path to it in parameters, I had to use ,exe example: ffmpeg_location = "/data/ffmpeg/bin/ffmpeg.exe", else it won't find it.

In the logs it's saying its stitching the video fine, but it doesn't save anything, then I get an exception that the mp4 file can't be found. All the png files in their folder do get saved, but the video file doesnt get saved in there, and I get the exception when the scipt goes back to bring video into the gui.

Thaks for reading my blog post lol.

**Comment by aivrar** [comment]
ok I'm back with updates here in 'Discussions' thread because I'm using Docker and stuff.

I found in the settings a place to enter he path location for ffmpeg, I never noticed or used that before, is it new? Anyway I've been trying to get this work properly entering a path location for ffmpeg and still have pronlems.

If I set it to '/data/ffmpeg/bin' I get error ~ PermissionError: [Errno 13] Permission denied: '/data/ffmpeg/bin'

If I set it to '/data/ffmpeg/bin/ffmpeg' I get file not found error.

If I set it directly to exe like so '/data/ffmpeg/bin/ffmpeg.exe' I don't get any error for ffmpeg, the logs tell me 'Video stitching done in 0.02 seconds!-1  | '

So it must be done, HOWEVER it's not, the video file does NOT get created at all. I'm checking text2vid.py file at all the paths, they look right to me,.

For Docker I have to hack the scripts a little to set the path to the model properly, that all works, this here with ffmpeg doesn't, and I don't think it's a bug it's just the scripts don't jive.

**Reply by B34STW4RS** [reply]
Recently had a similar issue with loopbackwave not being able to stitch video, and the problem turned out to be how my subfolders were set up for the images. Disabling all the subfolder save options in settings fixed it, but that was no solution.

**Comment by kabachuha** [comment]
@hithereai need your help with ffmpeg here
